### PR TITLE
Create CreateLocalNode page

### PR DESCRIPTION
**Pull request by IvanaAndersson**

### CODE REVIEW

**Review by HristoNedelkov — APPROVED** [review]
👍

File: src/pages/CreateLocalNodePage/CreateLocalNodePage.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    '#216266',
+                    '#F66F8F',
+                    '#C78F67',
+                    '#CB6256',
+                  ]}
+                  color={chosenColor}
+                  onChange={color => setChosenColor(color.hex)}
+                />
+              </YStack>
+            </XStack>
```
Comment (HristoNedelkov) [inline src/pages/CreateLocalNodePage/CreateLocalNodePage.tsx]: You can export this in another file so we can use it too.

Comment (IvanaAndersson) [reply src/pages/CreateLocalNodePage/CreateLocalNodePage.tsx]: thanks, working on it

**Review by HristoNedelkov — APPROVED** [review]
👍

**Review by GoNZooo — APPROVED** [review]

**Pull request merged**